docs/help/in/otr.in

```diff
@@ -0,0 +1,111 @@
+
+OTR %|[OPTION]
+
+Command to control the OTR module. Without an option, this help is printed.
+
+This help contains three sections which are %9options, quickstart and files.%n
```

Comment:
This help file is kinda massive, would be nice to split these into separate help topics.